### PR TITLE
Find by phone number

### LINKED ISSUE #134: As a tutor, I can find a student by his phone number
- Update find command to use `p/` flag

### PR DESCRIPTION
**Pull request by cadencjk**
This PR will:
- Find students using phone number. e.g. `find p/81234567`
- Add testing for the relevant find commands

Closes #134

### CODE REVIEW

**Comment by cadencjk** [comment]
@seanflyyy Might wanna consider this implementation for your #135

File: src/main/java/seedu/address/model/person/predicate/PhoneContainsNumberPredicate.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package seedu.address.model.person.predicate;
+
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.StringUtil;
+import seedu.address.model.person.Person;
+
+/**
+ * Tests that a {@code Person}'s {@code Class} matches the phone given.
```
Comment (sjoann) [inline src/main/java/seedu/address/model/person/predicate/PhoneContainsNumberPredicate.java:9]: I think this should be {@code Phone}

File: src/test/java/seedu/address/logic/commands/FindCommandTest.java [hunk 2]
```diff
@@ -61,12 +71,16 @@ public void equals() {
...
         assertFalse(findClassOneCommand.equals(null));
+        assertFalse(findPhoneOneCommand.equals(null));
 
         // different person -> returns false
         assertFalse(findFirstCommand.equals(findSecondCommand));
 
         // different date -> returns false
         assertFalse(findClassOneCommand.equals(findClassTwoCommand));
+
+        //different phone -> returns false
```
Comment (sjoann) [inline src/test/java/seedu/address/logic/commands/FindCommandTest.java:82]: can add a whitespace in front (minor stuff)

**Review by sjoann — APPROVED** [review]
LGTM! just some cosmetic changes

**Pull request merged**